README.md

```
@@ -1,8 +1,12 @@
 # File Explorer
 
+<p align="center">
+  <img src="assets/Icons/Logo.png" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)